xbmc/cores/AudioEngine/DSPAddons/ActiveAEDSP.cpp

```diff
@@ -75,36 +74,29 @@ CActiveAEDSP::CActiveAEDSP()
 
 CActiveAEDSP::~CActiveAEDSP()
 {
-  /* Deactivate all present dsp addons */
   Deactivate();
+  CAddonMgr::GetInstance().UnregisterAddonMgrCallback(ADDON_ADSPDLL);
+  CSettings::GetInstance().UnregisterCallback(this);
   CLog::Log(LOGDEBUG, "ActiveAE DSP - destroyed");
 }
-//@}
 
-/*! @name initialization and configuration methods */
-//@{
-class CActiveAEDSPStartJob : public CJob
+void CActiveAEDSP::Init(void)
 {
-public:
-  CActiveAEDSPStartJob() {}
-  ~CActiveAEDSPStartJob(void) {}
+  std::set<std::string> settingSet;
+  settingSet.insert(CSettings::SETTING_AUDIOOUTPUT_DSPSETTINGS);
```

Comment:
To be consistent would it be possible to rename `SETTING_AUDIOOUTPUT_DSPSETTINGS` to `SETTING_AUDIOOUTPUT_ADSPSETTINGS`?